src/current/v23.2/row-level-ttl.md

```diff
@@ -26,7 +26,9 @@ At a high level, Row-Level TTL works by:
 - Issuing batched [`DELETE`]({% link {{ page.version.version }}/delete.md %}) statements for the expired rows.
 - As part of the above process, deciding how many rows to [`SELECT`]({% link {{ page.version.version }}/select-clause.md %}) and [`DELETE`]({% link {{ page.version.version }}/delete.md %}) at once in each of the above queries.
 - Running the SQL queries described above in parallel as [background jobs]({% link {{ page.version.version }}/show-jobs.md %}).
-- To minimize the performance impact on foreground application queries, the background deletion queries are rate limited; they are also submitted at a lower priority level using the [admission control system]({% link {{ page.version.version }}/admission-control.md %}).
+- To minimize the performance impact on foreground application queries, the background deletion queries are rate limited; they are also submitted at a lower priority level using the [admission control system]({% link {{ page.version.version }}/admission-control.md %}). When foreground traffic increases, CockroachDB will reduce the resources allocated to TTL deletes to handle the foreground traffic. When foreground traffic decreases, CockroachDB will increase the resources allocated to TTL deletes.
+
+- {% include_cached new-in.html version="v23.2" %} Latency of row-level TTL queries is further reduced by using the elastic CPU limiter, which dynamically controls the total CPU percentage used by row-level TTL reads. The elastic CPU limiter can be disabled for row-level TTL queries by setting the `kvadmission.ttl_read_elastic_control.enabled` and `sqladmission.low_pri_read_response_elastic_control.enabled` [cluster settings]({% link {{ page.version.version }}/cluster-settings.md %}) to `false`.
```

Comment:
intentional that we are documenting non-public cluster settings here?  not seeing either of these show up in https://www.cockroachlabs.com/docs/v23.2/cluster-settings 

could just as easily get support issues from people turning these off and having problems, right?  my understanding is if cluster settings are not public/visible we don't support customers using them without Support team involved

meta comment (not fixable in a PR but for your consideration @dikshant) : 

do we have broad alignment with Eng on what cluster settings are public vs non-public, and what that actually means?  I feel like if we audited our docs for "non-public" cluster settings we would find A TON.  So at that point what even is the point of this public/non-public distinction? And does everyone align on what that distinction is? (end meta comment lol)